subcontracted_service/__openerp__.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+# Author: Damien Crier
+# Copyright 2017 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": 'Subcontracted service',
+    "summary": 'Subcontracted service',
+    "version": "9.0.1.0.0",
+    "category": "Purchase",
+    "website": "https://odoo-community.org/",
+    "author": "Camptocamp, Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "application": False,
+    "installable": True,
+    "depends": [
+        "purchase",
+        "sale_service",
```

Comment:
I think there is no real dependency on sale_service. Also this module has been removed in v10.